### PR TITLE
Add ADR for consolidating moves and MTOs tables

### PR DESCRIPTION
**Pull request by monfresh**
This ADR proposes consolidating moves and move_task_orders into one
moves table based on the assumption that a move can only have one MTO
at a given time. We would like input from everyone to make sure our
assumptions are valid.

## Description

Explain a little about the changes at a high level.

## Reviewer Notes

Is there anything you would like reviewers to give additional scrutiny?

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
echo "Code goes here"
```

## Code Review Verification Steps

* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/wiki/backend#logging)
* [ ] The requirements listed in
 [Querying the Database Safely](https://github.com/transcom/mymove/wiki/backend#querying-the-database-safely)
 have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/wiki/migrate-the-database#zero-downtime-migrations))
  * [ ] Have been communicated to #g-database
  * [ ] Secure migrations have been tested following the instructions in our [docs](https://github.com/transcom/mymove/wiki/migrate-the-database#secure-migrations)
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](tbd) for this change
* [this article](tbd) explains more about the approach used.

## Screenshots

If this PR makes visible UI changes, an image of the finished UI can help reviewers and casual
observers understand the context of the changes. A before image is optional and
can be included at the submitter's discretion.

Consider using an animated image to show an entire workflow instead of using multiple images. You may want to use GIPHY CAPTURE for this! 📸

_Please frame screenshots to show enough useful context but also highlight the affected regions._

### CODE REVIEW

**Comment by tinyels** [comment]
This looks good to me. I would possibly add a note indicating that there will be future work to reconcile the ppm fields on this table (`ppm_type` and `ppm_estimated_weight`) when we reenter conversations with the prime.

File: docs/adr/0053-consolidate-moves-and-mtos.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+anywhere in the app. This came out of a HackerOne engagement where hundreds of fake moves were created.
+
+* `contractor_id`: This was added to represent the prime contractor who will
+handle the move. This makes it easy to point the move to a different contractor in case it changes.
+
+* `available_to_prime_at`: a date and time field that indicates when the move is
+available for the Prime to handle. The presence of this field can be used to
+determine whether or not to display the move to the Prime.
+
+* `ppm_type`: currently used values are `FULL` and `PARTIAL`. This appears to be
```
Comment (shimonacarvalho) [inline docs/adr/0053-consolidate-moves-and-mtos.md:72]: Could we add NONE to `ppm_type` - not every move has a ppm.

File: docs/adr/0053-consolidate-moves-and-mtos.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+human-readable identifier for a move (so it could be disclosed to support staff,
+for instance). The MTO's `reference_id` is similar in nature but is in a `dddd-dddd`
+format. The `reference_id` does serve currently as the prefix for payment request
+numbers. We likely don’t need both `locator` and `reference_id` if these tables
+merge. See [Slack discussion](https://ustcdp3.slack.com/archives/CP6PTUPQF/p1595605700223400).
+
+* `cancel_reason`: A string to explain why a move was canceled.
+
+* `show`: A boolean that allows admin users to prevent a move from showing up
+anywhere in the app. This came out of a HackerOne engagement where hundreds of fake moves were created.
```
Comment (sandy-wright) [inline docs/adr/0053-consolidate-moves-and-mtos.md:63]: oh, this is interesting! This is mostly unrelated to the ADR, but I'd like to know more of what `show` does and it when it should be used. Is it also used to hide all sub-objects related to this move (shipments, service items, etc)? I'm wondering if we should start using this when we use the Support API to create moves during load testing.

Comment (monfresh) [reply docs/adr/0053-consolidate-moves-and-mtos.md:63]: As far as I can tell, all moves default to `show` being `true`, and any move that has `show` set to `false` will be excluded from the queue. I think the only expected way to set a move's `show` to `false` is via the admin interface. Here is the PR that updated the queue query to only include Moves that have `show` set to `true`: https://github.com/transcom/mymove/pull/1959/files

**Pull request merged**